sidebars.js

```diff
@@ -97,7 +97,8 @@ const sidebars = {
         'self-paced-labs/kops/index',
         'self-paced-labs/kwok-in-docker/index',
         'self-paced-labs/vault/index',
-        'self-paced-labs/quarkus-with-helm-charts/index'
+        'self-paced-labs/quarkus-with-helm-charts/index',
+        'self-paced-labs/vault-secret-store-csi-with-quarkus/index'
```

Comment:
按照上次 review 說明應為 Secret Store CSI，此故這裡改為 secrets
```suggestion
        'self-paced-labs/vault-secrets-store-csi-with-quarkus/index'
```